IBMigrationTool/main.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  main.swift
+//  IBMigrationTool
+//
+//  Created by Simon Mitchell on 17/09/2020.
+//  Copyright © 2020 threesidedcube. All rights reserved.
+//
+
+import Foundation
+
+extension FileManager {
+    
+    /// Recurses the given directory for files with a given extension
+    /// - Parameters:
+    ///   - type: The extension to find files for
+    ///   - directory: The directory to search in
+    /// - Returns: An array of file paths
+    func recursivePathsForResource(_ type: String?, directory: String) -> [String] {
+        
+        var filePaths: [String] = []
+        let enumerator = FileManager.default.enumerator(atPath: directory)
```

Comment:
As this in an extension of `FileManager` we probably want to remove `FileManager.default.`